src/cbmc/cbmc_solvers.h

```diff
@@ -68,13 +68,13 @@ class cbmc_solverst:public messaget
 
     prop_convt &prop_conv() const
     {
-      PRECONDITION(prop_conv_ptr!=nullptr);
+      PRECONDITION(prop_conv_ptr != nullptr);
```

Comment:
Ah, that seems to be fixing my clang-format comment placed on the earlier commit.